README.md

```diff
@@ -56,6 +58,40 @@ const errorHandler = (err: Error) => {
 const metricsClient = createStatsDClient(StatsD, config, errorHandler);
 ```
 
+### `createLambdaExtensionClient`
+
+`createLambdaExtensionClient` creates a [lambda extension](https://docs.datadoghq.com/serverless/libraries_integrations/extension/) client.
+This is intended for AWS Lambda functions and is a replacement for `createCloudWatchClient`.
```

Comment:
Can we update the deprecation notice on `createCloudWatchClient` to advise users to switch to `createLambdaExtensionClient`?